workspaces/archive/problems.jsonl

```diff
@@ -3325,3 +3325,13 @@
 {"chalengeQuestionDates":[],"difficulty":"Medium","isPaidOnly":false,"questionFrontendId":3325,"title":"Count Substrings With K-Frequency Characters I","titleSlug":"count-substrings-with-k-frequency-characters-i"}
 {"chalengeQuestionDates":[],"difficulty":"Medium","isPaidOnly":false,"questionFrontendId":3326,"title":"Minimum Division Operations to Make Array Non Decreasing","titleSlug":"minimum-division-operations-to-make-array-non-decreasing"}
 {"chalengeQuestionDates":[],"difficulty":"Hard","isPaidOnly":false,"questionFrontendId":3327,"title":"Check if DFS Strings Are Palindromes","titleSlug":"check-if-dfs-strings-are-palindromes"}
+{"chalengeQuestionDates":[],"difficulty":"Medium","isPaidOnly":true,"questionFrontendId":3328,"title":"Find Cities in Each State II","titleSlug":"find-cities-in-each-state-ii"}
```

Comment:
hmm why is `chalengeQuestionDates` missing an extra `l` 🤔

Reply:
Typo on my part 😅